### DISCUSSION TITLE
Questions about 'SampleTypeConfig'

### DISCUSSION THREAD

**Discussion by Jun10ng** [grafana/pyroscope#2422] [Q&A]
Hi @korniltsev 
I noticed that you’ve made some disruptive changes to the functionality of pushing pprof files in[ this issue.](https://github.com/grafana/pyroscope/pull/2376) You’ve introduced a sample_type_config and even save a related file on the machine. Could you please explain what exactly the sample_type_config file contains?

Thanks!

**Comment by cyriltovena** [comment]
Sample types is the type use for profiles sample. for CPU it's usually time, for memory it's bytes.
